src/models/block.rs

```diff
@@ -59,16 +59,16 @@ pub struct ExternalFileObject {
 #[serde(rename_all = "snake_case")]
 pub enum FileOrEmojiObject {
     Emoji { emoji: String },
-    File { file: InternalFileObject },
-    External { external: ExternalFileObject },
+    File,
+    External
```

Comment:
rust format is complaining that this doesn't have a comma. If you can satisfy the CI I can approve the review.